src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceKnownTypeTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+
+public static class ServiceKnownTypeTests
+{
+    [Fact]
+    [OuterLoop]
+    public static void ServiceKnownType_DataContract_AttrOnMethod_Test()
```

Comment:
I know this is annoying but could you add the comments we are now using with all new tests to call out the distinct sections of the test. One of the reasons for insisting on this is that we are hoping to use our test bed as examples for the community on how to use WCF and we want them to have a consistent style and syntax. See tests in other areas for guidance such as Scenarios\Contract\Service\ServiceContractTests.cs.
// *** SETUP *** \\
// *** EXECUTE *** \\
// *** VALIDATE *** \\
// *** CLEANUP *** \\
// *** ENSURE CLEANUP *** \\